concord-ui/package.json

```diff
@@ -3,16 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "react": "16.2.0",
-    "react-dom": "16.2.0",
+    "react": "16.5.2",
+    "react-dom": "16.5.2",
     "react-intl": "2.4.0",
     "react-redux": "5.0.7",
     "react-router-dom": "4.2.2",
     "react-scripts": "2.1.1",
     "react-select": "^2.1.2",
     "redux": "4.0.1",
     "redux-thunk": "2.3.0",
-    "rmwc": "3.0.10",
+    "rmwc": "4.0.6",
```

Comment:
v5.1.8 did not work

Reply:
What did not work ? It would be good to be on the latest version if possible...

Reply:
V5 didn't compile at all, needed to upgrade from v3 the DataTable. V4 seems to work.